datastore.go

```diff
@@ -224,6 +231,6 @@ func (d *Datastore) NewTransaction(readOnly bool) (ds.Txn, error) {
 	if err != nil {
 		return nil, err
 	}
-	accessor := &accessor{tx}
+	accessor := &accessor{ldb: tx, syncWrites: false}
```

Comment:
`false` was chosen here because it's the default behavior and since the internal transaction totally ignores `WriteOptions` anyway.